### ISSUE TITLE
Errors when reading the DELF pre-trained model

### ISSUE THREAD

**Issue by zhpcui** [tensorflow/models#2711]
**System information**
Linux Ubuntu 16.04
TensorFlow 1.4.0 (GPU support) installed using Anaconda
CUDA: 8.0 
cuDNN: 6.0
Python: 2.7
GPU: NVIDIA GeForce GTX 1080 Ti

**Describe the problem**
I successfully installed DELF by following the illustration (https://github.com/tensorflow/models/blob/master/research/delf/INSTALL_INSTRUCTIONS.md).
However when I tried the given example with pre-trained model (delf_v1_20171026), I got some errors as the following:
...
W tensorflow/core/framework/op_kernel.cc:1192] Data loss: Unable to read file (parameters/delf_v1_20171026/model/variables/variables.index). Perhaps the file is corrupt or was produced by a newer version of TensorFlow with format changes (failed to seek to header entry): corrupted compressed block contents
...

**Comment by zhpcui** [comment]
I have solved the problem by switching TF 1.4.0 to TF 1.3.0.

**Comment by andrefaraujo** [comment]
thanks for your continued investigation, @zhpcui! Sorry that you were having problems with this. I will update the delf repo to mention this issue.

**Comment by andrefaraujo** [comment]
I discussed with @vrv and this seems exactly the same problem as #2648. My understanding is that there were some problems in the TF 1.4.0 release, which are being worked on right now and will be fixed very soon.

**Comment by zhpcui** [comment]
Thank you very much, Andre.

**Comment by angersson** [comment]
This appears to be resolved (or on the track to resolution), so I'm closing this issue to keep the tracker focused. Feel free to reopen it if needed.
